workers/loc.api/sync/get-positions-snapshot/index.js

```diff
@@ -94,6 +95,50 @@ const _getPositionsHistoryIds = (positionsHistory) => {
     }, [])
 }
 
+const _convertPlToUsd = async (
+  rService,
+  pl,
+  symbol,
+  end
+) => {
+  const currency = symbol.slice(-3)
+  const _isForexSymb = isForexSymb(currency)
+
+  if (
+    _isForexSymb ||
+    currency.length < 3 ||
+    !Number.isFinite(pl)
+  ) {
+    return null
+  }
+
+  const reqSymb = `t${currency}USD`
+
+  const {
+    res: publicTrades
+  } = await rService._getPublicTrades({
+    params: {
+      reqSymb,
+      end,
+      limit: 1,
+      notThrowError: true,
+      notCheckNextPage: true
+    }
+  })
+
+  if (
+    !Array.isArray(publicTrades) ||
+    publicTrades.length === 0 ||
+    !publicTrades[0] ||
+    typeof publicTrades[0] !== 'object' ||
+    !Number.isFinite(publicTrades[0].price)
+  ) {
+    return null
+  }
+
+  return pl * publicTrades[0].price
```

Comment:
Why using pubTrades instead of candles?

Reply:
Reason: candles are not sync for positions, on next upgrade would sync candles of positions also. And change this.